Sum.py

```diff
@@ -0,0 +1,5 @@
+sum = 0
+for i in range(1, 101):
+    if i % 3 != 0 and i % 7 != 0 and i % 13 != 0:
+        sum = sum+i
```

Comment:
sum = sum+1 is not wrong but use pythonic way and write 
sum += 1

Reply:
okay sir,i will remember next time.